README.md

```diff
@@ -1,21 +1,15 @@
-Friendica Social Communications Server
-======================================
+Friendica - your open and free social network
+=============================================
 
-Welcome to the free social web.
+Welcome to the free social web. Friendica is a platform for decentralised social communication linking to other independent social and corporate services.
 
-Friendica is a decentralised communications platform that integrates social communication. Our platform links to independent social projects and corporate services.
+Being part of the Fediverse allows you to be free from data-harvesting corporations. Enjoy open social communication, independent of any specific provider.
```

Comment:
Please move this after the next paragraph since it now contains a rough definition of the Fediverse.

```suggestion
Being part of the Fediverse allows you to be free from data-harvesting corporations. Enjoy open social communication, independent of any specific provider.
```